tools/vp20compiler/nvvertparse.c

```diff
@@ -119,15 +119,16 @@ _mesa_find_line_column(const unsigned char *string, const unsigned char *pos,
  */
 static void
 record_error(struct parse_state *parseState, const char *msg, const char* file,
-             int lineNo)
+             int lineNo, bool onlyWarn)
 {
    int line, column;
    const unsigned char *lineStr;
    lineStr = _mesa_find_line_column(parseState->start,
                                     parseState->pos, &line, &column);
    // _mesa_debug(parseState->ctx,
```

Comment:
Why not also remove unused, out-comment'ed, undocumented code? Is there a reason to keep it?

Reply:
It's not touched by this PR, so I avoided touching it. Can be done in a cleanup PR later (I probably won't do it, because it's not really important and the code quality in this file isn't too great to begin with).